webapp/graphite/storage.py

```diff
@@ -8,10 +8,21 @@
 from django.conf import settings
 
 from graphite.util import is_local_interface, is_pattern
+from graphite.util import MAX_QUERY_LIMIT, MatchesOverflowError
 from graphite.remote_storage import RemoteStore
 from graphite.node import LeafNode
 from graphite.intervals import Interval, IntervalSet
 from graphite.readers import MultiReader
+from graphite.logger import log
+
+
+def send_to_carbon(metric, value):
```

Comment:
有 statsd 库的

On Wed, Jun 8, 2016, 1:40 PM WaySLOG <notifications@github.com> wrote:

> In webapp/graphite/storage.py
> <https://github.com/eleme/graphite-web/pull/3#discussion_r66195067>:
>
> >  from graphite.remote_storage import RemoteStore
> >  from graphite.node import LeafNode
> >  from graphite.intervals import Interval, IntervalSet
> >  from graphite.readers import MultiReader
> > +from graphite.logger import log
> > +
> > +
> > +def send_to_carbon(metric, value):
>
> udp发到statsd咯？
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/eleme/graphite-web/pull/3/files/f5c5d5b92c4055efc79c726713233f2ef1277fe5#r66195067>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ABFB6MsRHByP0AYAAVYs2Tpz9j6otN99ks5qJlWxgaJpZM4Ivwku>
> .
>